Jobsway2go.Core/Models/Role.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Jobsway2go.Core.Models
+{
+    public class Role:IdentityRole
+    {
+        public int Id { get; set; }
+        public string RoleName { get; set; }
+    }
+}
```

Comment:
Check if `Id` in `IdentityRole` is int, if not then remove only the `RoleName` property.